landoapi/hgexports.py

```diff
@@ -393,6 +393,147 @@ def get_timestamp(self) -> str:
         return get_timestamp_from_git_date_header(date)
 
 
+ACCEPTABLE_MESSAGE_FORMAT_RES = [
```

Comment:
I think it's a bit odd to have all these definitions (396:456) in the middle of this module. May be better in a different supporting module, or perhaps at the top. Could also define the ones that are ultimately used in the methods, in those methods that are using them, since it doesn't appear that anything is used in more than one method.

Reply:
Now that I look at these, they make more sense in `commit_message.py`.